File: auctionwebapp.client/src/app/simulationUser.ts
export class SimulationUser {
  constructor(
    public Id: number,
    public Color: string,
    public Name: string = "tester" + Id,
    public EstimatedCost: number = 0,
    public Budget: number = 0,
    public BetProbabilityBefore: number = 0,
    public BetProbabilityAfter: number = 0,
    public Q: number = 0
  ) { }
}

File: AuctionWebApp.Server/Data/Dto/SimulationUser.cs
namespace AuctionWebApp.Server.Data.Dto
{
    public class SimulationUser
    {
        public int Id { get; set; }
        public string Name { get; set; } = "user";
        public ulong EstimatedCost { get; set; }
        public ulong Budget { get; set; }
        public double BetProbabilityBefore { get; set; }
        public double BetProbabilityAfter { get; set; }
        public double Q { get; set; }
    }
}
